troubleshooting.md

```diff
@@ -0,0 +1,333 @@
+| ClusterBuilder     | kpack        | Cluster              | ClusterBuilders are kpack components installed by TAP that are used to build images. If they are not present, image builds may not complete successfully.                                                                                                                                                                                                    |
+| Deployment         | kubernetes   | `tas-adapter-system` | All of the Application Service Adapter Deployment's `Ready` counts should be listed as `1/1` except for `korifi-api-deployment` which should be `2/2`. Should this not be the case, the system is not functioning correctly and the status/events fields of the inoperable Deployment should be analyzed. This issue can sometimes be related to RBAC.       |
+| ClusterRole        | kubernetes   | Cluster              | ClusterRoles are created to manage system privileges at the cluster scope.                                                                                                                                                                                                                                                                                   |
+| ClusterRoleBinding | kubernetes   | Cluster              | ClusterRoleBindings are created to manage system privileges at the cluster scope.                                                                                                                                                                                                                                                                            |
+| ServiceAccount     | kubernetes   | `cf`                 | ServiceAccounts are created to manage CF permissions. Some ServiceAccounts are designed to be propagated to CFOrg/CFSpace namespaces. These objects will be designated with the `cloudfoundry.org/propagate-cf-role: "true"` annotation.                                                                                                                     |
+| RoleBinding        | kubernetes   | `cf`                 | RoleBindings are created to manage CF permissions. Some RoleBindings are designed to be propagated to CFOrg/CFSpace namespaces. These objects will be designated with the `cloudfoundry.org/propagate-cf-role: "true"` annotation.                                                                                                                           |
+| SCC                | openshift    | Cluster              | SCCs are specific to OpenShift and are used to grant required permissions to Application Service Adapter components in that environment. These CRDs do not exist in non-OpenShift environments. An associated ClusterRole, ClusterRoleBinding, and RoleBinding will also be installed if the `kubernetes_distribution` shared setting is set to `openshift`. |
```

Comment:
is the actual kind for this SCC or is it `SecurityContextConstraints`? I think we should define it using its full name at least once for clarity.

Reply:
It looks like SCC was a shortcut so chaned to SecurityContextConstraints.